sdk/trace/src/test/java/io/opentelemetry/sdk/trace/SdkSpanBuilderTest.java

```diff
@@ -58,14 +58,16 @@
           TraceState.getDefault());
   private final SpanProcessor mockedSpanProcessor = Mockito.mock(SpanProcessor.class);
 
-  private final SdkTracerProvider tracerSdkFactory = SdkTracerProvider.builder().build();
-  private final SdkTracer sdkTracer = (SdkTracer) tracerSdkFactory.get("SpanBuilderSdkTest");
+  private SdkTracerProvider tracerSdkFactory;
+  private SdkTracer sdkTracer;
 
   @BeforeEach
   public void setUp() {
+    tracerSdkFactory = SdkTracerProvider.builder().addSpanProcessor(mockedSpanProcessor).build();
```

Comment:
Need to initialize fields that use mocks in `BeforeEach` so they run after mockito. It's a reason why I almost never initialize fields inline in test classes.